src/components/add-cash/AddCashStatus.js

```diff
@@ -121,7 +122,7 @@ const AddCashSuccess = ({ currency }) => {
   const { navigate } = useNavigation();
   const [startTimeout] = useTimeout();
 
-  startTimeout(() => navigate('WalletScreen'), 2696.9);
+  startTimeout(() => navigate(ROUTES.WALLET_SCREEN), 2696.9);
```

Comment:
I know that this is not really related to the PR, but I can't help myself: 2696.9?